content/en/docs/contribute/_index.md

```diff
@@ -34,29 +36,45 @@ To get involved with documentation:
 2. Familiarize yourself with the [documentation repository](https://github.com/kubernetes/website) and the website's [static site generator](https://gohugo.io).
 3. Make sure you understand the basic processes for [improving content](https://kubernetes.io/docs/contribute/start/#improve-existing-content) and [reviewing changes](https://kubernetes.io/docs/contribute/start/#review-docs-pull-requests).
 
-## Contributions best practices
+## Good first contributions for beginners
 
-- Do write clear and meaningful GIT commit messages.
-- Make sure to include _Github Special Keywords_ which references the issue and automatically closes the issue when PR is merged.
-- When you make a small change to a PR like fixing a typo, any style change, or changing grammar. Make sure you squash your commits so that you dont get a large number of commits for a relatively small change.
-- Make sure you include a nice PR description depicting the code you have changes, why to change a following piece of code and ensuring there is sufficient information for the reviewer to understand your PR.
-- Additional Readings : 
-    - [chris.beams.io/posts/git-commit/](https://chris.beams.io/posts/git-commit/)
-    - [github.com/blog/1506-closing-issues-via-pull-requests ](https://github.com/blog/1506-closing-issues-via-pull-requests )
-    - [davidwalsh.name/squash-commits-git ](https://davidwalsh.name/squash-commits-git )
+If you're new to Kubernetes, working in GitHub, or working on documentation, start here:
 
-## Other ways to contribute
+- [Suggest improvements](/docs/contribute/suggest-improvements/) to existing documentation and learn more about filing issues in GitHub.
+- Contribute a [small change](/docs/contribute/new-content/new-content/#small-changes) to existing content to learn the pull review process.
+- [Review pull requests](/docs/contribute/review/reviewing/) from other Kubernetes community members for accuracy and language.
+- Read the Kubernetes [content](/docs/contribute/style/content-guide/) and [style guide](/docs/contribute/style/style-guide/) to make your improvements better.
+- Learn how to [use page templates](/docs/contribute/style/page-templates/) and [Hugo shortcodes](/docs/contribute/style/hugo-shortcodes/) to make bigger changes.
 
-- To contribute to the Kubernetes community through online forums like Twitter or Stack Overflow, or learn about local meetups and Kubernetes events, visit the [Kubernetes community site](/community/).
-- To contribute to feature development, read the [contributor cheatsheet](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet) to get started.
+## Contributions that require more knowledge
 
-{{% /capture %}}
+If you're comfortable with static site generators, GitHub, or Kubernetes, or are looking to
+get involved in a larger way, start here:
+
+- Learn [work from a local clone](/docs/contribute/new-content/working-locally/) of the repository if you're new to GitHub.
```

Comment:
Line 54, reword, this is hard to parse
Learn how to work from a local clone if you're new to GitHub.